### DISCUSSION TITLE
Visualize a given RA/DEC equatorial coordinate

### DISCUSSION THREAD

**Discussion by gerben-b** [cosinekitty/astronomy#152] [Q&A] [answered]
Hi Don,

Some time ago I created moon_cam using your Astronomy engine (javascript). Currently I am creating an advanced version with a built-in planetarium. Most of it is working now and i will publish that soon on github, but one thing is missing: I would like to plot a given equatorial coordinate on my screen and for that I need to transform given RA/DEC (equatorial) coordinates into ALT/AZ (horizontal) coordinates. This means that the coordinates for this purpose cannot be related to a celestial body.

Can you give me some pointer on how to do this with your engine?

Thanks,
Gerben

**Comment by cosinekitty** [comment, marked as answer]
Hi @gerben-b!  I'm glad to hear you are making good progress on your project. Yes, what you want to use is the [Horizon](https://github.com/cosinekitty/astronomy/tree/master/source/js#Horizon) function. It does exactly what you describe. It optionally corrects for atmospheric refraction, which I suspect you will want.

**Reply by gerben-b** [reply]
Thanks, that is indeed exactly what I need. The first thing that I want to do is display a cross hair on my planetarium. That will help me to better understand where I am in relation to e.g. the moon.

Also I noticed you have the Gallilean moons of Jupiter now in your engine, very nice indeed :-) Thanks for everything.

**Reply by cosinekitty** [reply]
When you get your project up on GitHub, please let me know. I would love to take a look at it!
